en/lessons/libraries/mox.md

```diff
@@ -155,22 +155,18 @@ Install it with `mix deps.get`.
 
 Next, modify your `test_helper.exs` so it does 3 things:
```

Comment:
Now this should read `Next, modify your `test_helper.exs` so it does *2* things:` 😄

Reply:
Done. Thanks for catching that @SophieDeBenedetto :)